### PR TITLE
adds new readme by @tomekwi

### PR DESCRIPTION
**Pull request by stoeffel**
Lets get started on #62.
I added the state we had in the issue. Feel free to change it (@tomekwi, @hemanth).

:no_entry_sign: not ready to merge

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,8 +12,46 @@
...
-> Useful oneliners and shorthand functions
-
+<sup>MADE IN SWITZERLAND</sup>
+
+1-liners
+====
+
+**Functional tools that couldn’t be simpler.**
+
+We’re proud to present *1-liners* – a stupid-simple functional utility belt. 62 *[@stoeffel this can update automatically]* one-liner functions, hand-crafted with love and attention.
```
Comment (stoeffel) [inline README.md]: should we move the README to a README.tpl?

File: README.md [hunk 2]
```diff
@@ -12,8 +12,46 @@
...
+> – the docs show usage and the source, and link to the specification. [documentation](https://github.com/1-liners/1-liners/tree/master/documentation)
+> 
+> 9. We shall provide functional versions of **native object methods**.
+> – like `array.reduce`.
+> 
+> 10. We shall provide functional versions of **language constructs**.
+> – like `+` or `&&`.
 
 
 ## Usage
```
Comment (stoeffel) [inline README.md]: should we move this to the documentation?

File: README.md (hunk 1 above)
Comment (hemanth) [inline README.md]: India as well ;)

Comment (tomekwi) [inline README.md]: Yeah. Switzerland, India, Poland and Germany to be exact :)

It just sounds good that way :) It reads: “This is a quality product developed with obsessive attention to detail.”

Comment (stoeffel) [inline README.md]: > It just sounds good with Switzerland

:bowtie:

Comment (tomekwi) [inline README.md]: Wow! Can we use this graphic @hemanth? Where does it come from?

Comment (stoeffel) [inline README.md]: @hemanth Awesome!!

Comment (hemanth) [inline README.md]: http://justinmezzell.tumblr.com/

Comment (tomekwi) [inline README.md]: :+1: :+1:

Let’s put this before/under the “decalogue”! What do you think, guys?

Comment (stoeffel) [inline README.md]: yop that would be great. Did you ask him?

Comment (tomekwi) [inline README.md]: @hemanth did

Comment (stoeffel) [inline README.md]: nice didn't see that. Thanks @hemanth.

Comment (hemanth) [inline README.md]: waiting for his reply...

Comment (hemanth) [inline README.md]: @stoeffel @tomekwi There we go, we got a [green](https://twitter.com/JustinMezzell/status/601843459570409472) :)

Comment (tomekwi) [inline README.md]: :clap:

**Comment by stoeffel** [comment]
//cc @hemanth @tomekwi 
I updated the readme and added the swissknife.
There is a new util which updates the function count in the readme.

**Comment by hemanth** [comment]
@stoeffel Let's write one ;)?

**Comment by stoeffel** [comment]
@hemanth I did : https://github.com/1-liners/1-liners/pull/86/files#diff-4fa497b88f9d301decb6c1710ccfa9b9

**Comment by stoeffel** [comment]
What do you think guys is this good enough to merge? ( for a start)

**Comment by hemanth** [comment]
:+1: :shipit:

**Comment by tomekwi** [comment]
:+1:

I still wanted to do some finishing touches. Reformat and reword some stuff. But I’ll add a separate pull request for that when I find some time.

  :rocket:
:boom:

**Comment by stoeffel** [comment]
I will rebase tomorrow.

**Comment by stoeffel** [comment]
merged manually

**Pull request closed without merging**